source/supported-hardware/cml-rvp.rst

```diff
@@ -0,0 +1,125 @@
+.. _comet-lake-rvp:
+
+Comet Lake CRB Boards
+-----------------------
+
+.. note:: 10th Generation Intel® Core Processor is formally known as |Comet Lake S|.
+
+Supported Boards
+^^^^^^^^^^^^^^^^^^^^^
+
+|SPN| supports **CML S RVP with PCH H, CML S RVP with PCH V, CML S RVP with H420E** platform.
+
+
+Building
+^^^^^^^^^^
+
+To build |SPN| for |CML| platform::
+
+    python BuildLoader.py build cml
+
+Replace ``cml`` with ``cmlv`` for PCH V.

```

Comment:
How about H420E ?  How to build ?

Reply:
I've modified the build instruction part by showing different build command for each PCH.